nifi-nar-bundles/nifi-graph-bundle/nifi-neo4j-cypher-service/pom.xml

```diff
@@ -79,7 +79,7 @@
         <dependency>
             <groupId>org.neo4j.driver</groupId>
             <artifactId>neo4j-java-driver</artifactId>
-            <version>1.6.2</version>
+            <version>4.3.4</version>
```

Comment:
Should we bump this to 4.3.7 or are there compatibility issues?

Reply:
I'll bump.